tika-core/src/main/java/org/apache/tika/parser/ParseContext.java

```diff
@@ -48,6 +48,10 @@
  */
 public class ParseContext implements Serializable {
 
+    /** Flag to invoke Text to TagRatio Parser */
+    
+    public static final boolean isTagRatioSet = false;
```

Comment:
No. Read ParseContext. It's a container of keys. You don't need isTagRatioSet as a variable. You need to set a key e.g., org.apache.tika.parser.text.tagRatio and then look for that key in the parse context.